.github/workflows/build.yml

```diff
@@ -245,7 +245,6 @@ jobs:
       matrix:
         arch: ["arm", "arm64", "386", "amd64"]
     env:
-      repo: ${{ github.event.repository.name }}
```

Comment:
This looks good to me . To confirm the issue here is that this event information is not available via scheduled actions, correct?

Reply:
Yes! That's it. On workflow_call events, the github.event.repository.name isn't available, so we need to reference something else.